wiki/Tournaments/CWC/2019/en.md

```diff
@@ -89,9 +89,87 @@ The osu!catch World Cup 2019 is run by various community members by distributing
 
 ---
 
+## Match Schedule: Group Stage
+
+### Saturday, 29. June 2018
+
+| Team A |  |  | Team B | Match Time (UTC) | Local Time A | Local Time B | |
+| ---: | ---: | :--- | :--- | :---: | :---: | :---: | :---: |
+| Hong Kong | ![][flag_HK] | ![][flag_KR] | South Korea | **11:00 UTC** | 19:00 UTC+08 | 20:00 UTC+09 |
+| Japan | ![][flag_JP] | ![][flag_CN] | China | **11:00 UTC** | 20:00 UTC+09 | 19:00 UTC+08 |
+| Sweden | ![][flag_SE] | ![][flag_GB] | Vietnam | **11:00 UTC** | 13:00 UTC+02 | 18:00 UTC+07 |
+| Taiwan | ![][flag_TW] | ![][flag_PH] | Philippines | **12:00 UTC** | 20:00 UTC+08 | 20:00 UTC+08 |
+| Netherlands | ![][flag_NL] | ![][flag_CN] | China | **12:00 UTC** | 14:00 UTC+02 | 20:00 UTC+08 |
+| Japan | ![][flag_JP] | ![][flag_AU] | Australia | **12:00 UTC** | 21:00 UTC+09 | 22:00 UTC+010 |
+| Norway | ![][flag_NO] | ![][flag_TW] | Taiwan | **13:00 UTC** | 15:00 UTC+02 | 21:00 UTC+08 |
+| Czech Republic | ![][flag_CZ] | ![][flag_NO] | Norway | **13:00 UTC** | 14:00 UTC+01 | 15:00 UTC+02 |
+| Australia | ![][flag_AU] | ![][flag_CN] | China | **13:00 UTC** | 23:00 UTC+010 | 21:00 UTC+08 |
+| Belgium | ![][flag_BE] | ![][flag_FI] | Finland | **14:00 UTC** | 15:00 UTC+01 | 17:00 UTC+03 |
+| Czech Republic | ![][flag_CZ] | ![][flag_PH] | Philippines | **14:00 UTC** | 15:00 UTC+01 | 22:00 UTC+08 |
+| Colombia | ![][flag_CO] | ![][flag_GB] | Vietnam | **14:00 UTC** | 10:00 UTC-04 | 21:00 UTC+07 |
+| Brazil | ![][flag_BR] | ![][flag_RU] | Russian Federation | **16:00 UTC** | 13:00 UTC-03 | 21:00 UTC+05 |
+| Romania | ![][flag_RO] | ![][flag_GB] | United States | **16:00 UTC** | 18:00 UTC+02 | 12:00 UTC-04 |
+| Venezuela | ![][flag_GB] | ![][flag_FI] | Finland | **17:00 UTC** | 13:00 UTC-04 | 20:00 UTC+03 |
+| Italy | ![][flag_IT] | ![][flag_MX] | Mexico | **17:00 UTC** | 19:00 UTC+02 | 12:00 UTC-05 |
+| Russian Federation | ![][flag_RU] | ![][flag_GB] | United States | **17:00 UTC** | 22:00 UTC+05 | 13:00 UTC-04 |
+| Belgium | ![][flag_BE] | ![][flag_GB] | Venezuela | **18:00 UTC** | 19:00 UTC+01 | 14:00 UTC-04 |
+| Canada | ![][flag_CA] | ![][flag_IT] | Italy | **18:00 UTC** | 14:00 UTC-04 | 20:00 UTC+02 |
+| Mexico | ![][flag_MX] | ![][flag_CL] | Chile | **18:00 UTC** | 13:00 UTC-05 | 14:00 UTC-04 |
+| Canada | ![][flag_CA] | ![][flag_CL] | Chile | **20:00 UTC** | 16:00 UTC-04 | 16:00 UTC-04 |
+| Sweden | ![][flag_SE] | ![][flag_CO] | Colombia | **20:00 UTC** | 22:00 UTC+02 | 16:00 UTC-04 |
+| Brazil | ![][flag_BR] | ![][flag_GB] | United States | **21:00 UTC** | 18:00 UTC-03 | 17:00 UTC-04 |
```

Comment:
GB but United States? I don't know which is correct
More occurences above and below, some are in Vietnam. (check line 109)